dev/build/windows/makecoq_mingw.sh

```diff
@@ -1136,7 +1207,7 @@ function copy_coq_license {
     install -D README.md                      "$PREFIXCOQ/license_readme/coq/ReadMe.md"
     install -D CHANGES                        "$PREFIXCOQ/license_readme/coq/Changes.txt"
     install -D INSTALL                        "$PREFIXCOQ/license_readme/coq/Install.txt"
-    install -D doc/README.md                  "$PREFIXCOQ/license_readme/coq/ReadMeDoc.md"
+    install -D doc/README.md                  "$PREFIXCOQ/license_readme/coq/ReadMeDoc.md" || true
```

Comment:
I'd like to insist that this is not appropriate for the Windows build scripts to install more files than the normal install targets do. If you feel like these files should be installed, please open a PR modifying the Makefile instead and remove these lines from here. There is no reason why these files should be installed on Windows but not on macOS nor Linux.

Reply:
@Zimmi48 : can we do this cleanup as a separate step? There is more to it and I don't want to make it too complicated. I will do this soon, but I don't want to touch the make files in this step as well. I think it would be cleaner to have a separate issue where changes here and the make file are done in a synchronized and thorough way.

My next steps in the pipeline regarding the installer are:

1.) Windows development environment (separate installation of Cygwin and compilation, support in place compilation, support opam based builds, support cygwin and MinGW builds)

2.) Switch from NSIS to WIX (Windows Installer XML Toolkit). WIX easily supports downloadable components and I just like MSI installers more than exe installers.

3.) Cleanup - I could do the cleanup of readme and license files at any time before - the additional addons also require some thoughts here. Another cleanup step is either completely removing (as absolete) or factoring out the support for old versions.

Reply:
Yes.